packages/devtools_app/test/standalone_ui/ide_shared/property_editor_test.dart

```diff
@@ -2,80 +2,145 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
+
 import 'package:devtools_app/devtools_app.dart';
-import 'package:devtools_app/src/standalone_ui/ide_shared/property_editor/property_editor_sidebar.dart';
+import 'package:devtools_app/src/service/editor/api_classes.dart';
+import 'package:devtools_app/src/standalone_ui/ide_shared/property_editor/property_editor_controller.dart';
+import 'package:devtools_app/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart';
 import 'package:devtools_app_shared/ui.dart';
 import 'package:devtools_app_shared/utils.dart';
 import 'package:devtools_test/devtools_test.dart';
 import 'package:devtools_test/helpers.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
 
 void main() {
-  const propertyEditor = PropertyEditorSidebar();
+  final eventController = StreamController<ActiveLocationChangedEvent>();
+  final eventStream = eventController.stream;
 
-  group('Property Editor input types', () {
-    setUpAll(() {
-      setGlobal(ServiceConnectionManager, FakeServiceConnectionManager());
-      setGlobal(IdeTheme, IdeTheme());
-    });
+  final locationToArgsResult =
+      <(TextDocument, CursorPosition), EditableArgumentsResult>{
```

Comment:
use named parameters in this Pattern so that we don't have to refer to them using $1 and $2 below